app/client/ui/AdminPanel.ts

```diff
@@ -78,9 +78,38 @@ export class AdminPanel extends Disposable {
   }
 
   private _buildMainContent(owner: MultiHolder) {
+    // If probes are available, show the panel as normal.
+    // Otherwise say it is unavailable, and describe a fallback
+    // mechanism for access.
     return cssPageContainer(
       dom.cls('clipboard'),
       {tabIndex: "-1"},
+      dom.maybe(this._checks.probes, probes => {
+        return probes.length > 0
+            ? this._buildMainContentForAdmin(owner)
+            : this._buildMainContentForOthers(owner);
+      }),
+      testId('admin-panel'),
+    );
+  }
+
+  /**
+   * Show something helpful to those without access to the panel,
+   * which could include a legit adminstrator if auth is misconfigured.
+   */
+  private _buildMainContentForOthers(owner: MultiHolder) {
+    return dom.create(AdminSection, t('Administrator Panel Unavailable'), [
+      dom('p', `You do not have access to the administrator panel.
+Please log in as an administrator.`),
+      dom('p', `Or, as a fallback, you can set:`),
+      dom('pre', 'GRIST_BOOT_KEY=secret'),
+      dom('p', ` in the environment and visit: `),
+      dom('pre', `/admin?key=secret`)
```

Comment:
Perhaps something like this, to make it easier to translate?

```suggestion
      dom('p', t(`You do not have access to the administrator panel.
Please log in as an administrator.`)),
      dom('p', t(`Or, as a fallback, you can set: {{bootKey}} in the environment and visit: {{url}}`), {bootKey: dom('pre', 'GRIST_BOOT_KEY=secret'), url: dom('pre', `/admin?key=secret`)}),
```